### DISCUSSION TITLE
How to modify the storefront and dashboard design, dashboard translations, permission settings.

### DISCUSSION THREAD

**Discussion by Fliper05** [saleor/saleor#9828] [Deployment] [answered]
Hello there,
I want to configure the design of the dashboard and storefront app. I would love to change the colors, but I am having a very hard time finding any CSS/HTML files to do these changes. Is there a way to do it through the dashboard?

Another inquiry I have is about the permission settings in themselves. I want to make it so that the main page is only accessible when the user is logged in.

Also, is there a way to have 2 different groups of users? As some customers of ours are small, therefore we want to give them different prices, while our wholesale customers should have better pricing. Is there any way how I could go about setting these up?

And finally, my last question is about the translations. The translations of the dashboard are incomplete, and I'd love to modify these so that they are complete on both the storefront and the dashboard.

Thank you so much for all your help :)

**Comment by wileyknight** [comment, marked as answer]
The storefront uses tailwind. You can override the css from a global css like normal.
